config/application.rb

```diff
@@ -97,6 +97,17 @@ class Application < Rails::Application
     end
 
     # Register Spree calculators
+    Rails.application.reloader.to_prepare do
+      Openfoodnetwork::Application.config.spree.calculators.shipping_methods = [
```

Comment:
Nice :tada:

I think this means we can delete the duplicate line in the initializer below. And we probably can/should just pull all the lines in `initializer 'spree.register.calculators'` up into this block (and then delete `initializer 'spree.register.calculators'`). What do you think?

Reply:
Yeah I was treading lightly but I think that makes sense... I'll push that and see if anything breaks :)

Reply:
Cool. We should probably add checking of calculators on shipping methods and payment methods etc to the test notes. Zeitwerk and Single Table Inheritance (which we use on calculators) is a notorious pain point...

Reply:
👍 Updated the description, I think this can go to Test Ready now